app/controllers/api/votes_controller.rb

```diff
@@ -4,15 +4,8 @@ class API::VotesController < API::RestfulController
   alias :update :create
 
   def my_votes
-    @votes = if params[:discussion_id]
-      load_and_authorize :discussion
-      @discussion.votes.includes({motion: [:author, :outcome_author]}, :user).for_user(current_user).most_recent
-    elsif params[:proposal_ids]
-      ids = params[:proposal_ids].split(',').map(&:to_i)
-      current_user.votes.where(motion_id: ids)
-    else
-      current_user.votes.most_recent.since(3.months.ago).most_recent
-    end
+    load_and_authorize :discussion
```

Comment:
Good question. I think so, but will test it manually; was actually hoping that an e2e test would catch something related to this, but they went green.